objects_test.go

```diff
@@ -108,6 +115,24 @@ func (s *ObjectsSuite) TestBlobHash(c *C) {
 	c.Assert(string(data), Equals, "FOO")
 }
 
+func (s *ObjectsSuite) TestBlobDecodeEncodeIdempotent(c *C) {
+	var objects []*core.MemoryObject
+	obj := &core.MemoryObject{}
+	obj.Write([]byte("foo"))
+	obj.SetType(core.BlobObject)
+	obj.Hash()
+	objects = append(objects, obj)
+	for _, object := range objects {
```

Comment:
there is only on object in `objects`, what is the point of having a slice of only one element?